src/dune_rules/ocaml_flags.ml

```diff
@@ -25,10 +27,28 @@ let dev_mode_warnings =
     |> List.rev_map ~f:wrange_to_flag
     |> String.concat ~sep:""
   in
-  let pre_3_3 = lazy (warnings_range all) in
-  let post_3_3 = lazy (warnings_range (Int.Set.union all (Int.Set.of_list [ 67; 69 ]))) in
+  let and_warnings lazy_range =
+    lazy_range,
+    lazy (warnings_range (Lazy.force lazy_range)) in
+  let range_pre_3_3, pre_3_3 =
+    and_warnings @@ lazy all in
+  let range_pre_3_13, pre_3_13 =
+    and_warnings @@ lazy (
+      Lazy.force range_pre_3_3
+      |> add 67
+      |> add 69
+    ) in
+  let range_later, later =
```

Comment:
CI says that `range_later` isn't used anywhere

Reply:
Fixed, thanks!